show.go

```diff
@@ -218,3 +218,21 @@ func (c *Client) SaveShowsForCurrentUser(ctx context.Context, ids []ID) error {
 
 	return c.execute(req, nil, http.StatusOK)
 }
+
+// GetEpisode gets an episode from a show.
+// API reference: https://developer.spotify.com/documentation/web-api/reference/get-an-episode
+func (c *Client) GetEpisode(ctx context.Context, id string, opts ...RequestOption) (*EpisodePage, error) {
```

Comment:
It seems odd for the return type here to be `EpisodePage`, do we not have an episode type ?

Reply:
`EpisodePage` is the episode type, it's just unfortunately named. The page type itself is `SimpleEpisodePage`